x-pack/plugins/cases/docs/cases_client_typedoc.json

```diff
@@ -2,24 +2,26 @@
   "entryPoints": [
     "../server/client/client.ts",
     "../server/client/typedoc_interfaces.ts",
-    "../server/client/attachments",
+    "../server/client/attachments/client.ts",
     "../server/client/cases/client.ts",
     "../server/client/cases/get.ts",
     "../server/client/cases/push.ts",
     "../server/client/configure/client.ts",
+    "../server/client/metrics/client.ts",
     "../server/client/stats/client.ts",
-    "../server/client/sub_cases/client.ts",
     "../server/client/user_actions/client.ts"
   ],
+  "entryPointStrategy": "expand",
   "exclude": [
     "**/mock.ts",
     "../server/client/cases/+(mock.ts|utils.ts|utils.test.ts|types.ts)"
   ],
   "excludeExternals": true,
   "out": "cases_client",
   "theme": "markdown",
-  "plugin": "typedoc-plugin-markdown",
-  "entryDocument": "cases_client_api.md",
```

Comment:
`typedoc` no longer supports this option. So we can't rename the generated markdown file.